dashboard/package.json

```diff
@@ -51,8 +51,15 @@
     "lint-fix": "tslint -c ./tslint.ci.json --fix 'src/**/*.{ts,tsx}'"
   },
   "lint-staged": {
-    "*.{ts,tsx}": ["tslint -c ./tslint.ci.json --fix", "prettier --write", "git add"],
-    "*.{js,json,css,md}": ["prettier --write", "git add"]
+    "*.{ts,tsx}": [
```

Comment:
I honestly don't know when or why this has changed, I guess that it is part of the `yarn add` ?